README.md

```diff
@@ -43,6 +43,11 @@ For detailed information on how to build and run the samples, check out our `REA
 [sentry-samples](https://github.com/getsentry/sentry-kotlin-multiplatform/tree/main/sentry-samples)
 folder.
 
+## Apple Privacy Manifest
+
+Starting with [May 1st 2024](https://developer.apple.com/news/?id=3d8a9yyh), iOS apps are required to declare approved reasons to access certain APIs. This also includes third-party SDKs.
+If you are using the Sentry Kotlin Multiplatform SDK with Apple device targets then update your Sentry Cocoa SDK to `8.21.0` minimum. However we recommend using `8.25.0` to ensure compatibility with Apple's guidelines.
```

Comment:
```suggestion
If you are using the Sentry Kotlin Multiplatform SDK with Apple device targets then update your Sentry Cocoa SDK to `8.25.0` minimum.
```

Reply:
let's make it as easy as possible for our users :) if they can update to 8.21.0, they should have no problem updating to 8.25.0 as well. we'll also update the docs

Reply:
another thought - let's add [docs](https://docs.sentry.io/platforms/flutter/data-management/apple-privacy-manifest/)+[troubleshooting](https://docs.sentry.io/platforms/flutter/troubleshooting/#missing-api-declaration-after-app-store-review) pages for KMP and link to them here?

Reply:
https://github.com/getsentry/sentry-docs/pull/9872